packages/shared/src/hooks/feed/useCardCover.tsx

```diff
@@ -26,11 +26,12 @@ export const useCardCover = ({
   onShare,
   className = {},
 }: UseCardCoverProps): UseCardCover => {
-  const { shouldShowOverlay, onInteract } = usePostShareLoop(post);
+  const { shouldShowOverlay, onInteract, currentInteraction } =
+    usePostShareLoop(post);
   const shouldShowReminder = useBookmarkReminderCover(post);
```

Comment:
We can now remove this line and have it coming from the hook.

Reply:
Sorry, this exact line 31. Then from the hook, we return `shouldShowReminder` from inside of it and be used here.

Reply:
got it.